src/runtime/basic-runtime-test.js

```diff
@@ -186,6 +186,49 @@ describes.realWin('installBasicRuntime', {}, (env) => {
       expect(basicSubscriptions).to.have.property(name);
     }
   });
+
+  describe('default onEntitlementsResponse', () => {
+    let entitlements;
+    let configuredCallback;
+
+    beforeEach(() => {
+      sandbox
+        .stub(BasicRuntime.prototype, 'setOnEntitlementsResponse')
+        .callsFake((callback) => {
+          configuredCallback = callback;
+        });
+      entitlements = {
+        consume: sandbox.stub(),
+        clone: () => entitlements,
```

Comment:
Could we remove this `clone` method? Looks like the tests never call it, so it's reducing the coverage %
https://app.codecov.io/gh/subscriptions-project/swg-js/compare/2106/diff

Reply:
Whoops, yeah. Removed. Had left that before I decided to just stub out and capture the configured callback